jadx-gui/src/main/resources/i18n/Messages_en_US.properties

```diff
@@ -15,6 +15,7 @@ menu.text_search=Text search
 menu.class_search=Class search
 menu.comment_search=Comment search
 menu.goto_main_activity=Go to main Activity
+menu.go_to_application=Go to Application
```

Comment:
Having one label with `goto` and one with `go_to` isn't really nice:
```
menu.goto_main_activity=...
menu.go_to_application=...
```

Reply:
I fixed it in a new commit. Renamed "goto" to "go_to" since it follows the pattern better.